C#/中文git/Program.cs

```diff
@@ -0,0 +1,250 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace DuckStudio.ChineseGit
+{
+    public class Program
+    {
+        // 获取程序路径
+        public static readonly string ScriptPath = Path.GetDirectoryName(Environment.GetCommandLineArgs()[0]);
+
+        // 中文Git完整路径
+        public static readonly string FullPath = Path.Combine(ScriptPath, "中文git.exe");
+
+        // 定义版本号
+        public const string Version = "v2.9.2-pack";
+
+        // 配置文件路径
+        public static readonly string configFile = Path.Combine(Path.GetDirectoryName(System.Reflection.Assembly.GetEntryAssembly().Location), "config.json");
+
+        static void Main()
+        {
+            Environment.ExitCode = 0;
+            bool autoCheckUpdate;
+            bool autoGetNotice;
+
+            if (File.Exists(configFile))
+            {
+                try
+                {
+                    // 读取配置文件内容并反序列化为对象
+                    string configJson = File.ReadAllText(configFile);
+                    var configData = JsonConvert.DeserializeObject<ConfigData>(configJson);
+
+                    // 获取自动检查更新和自动获取通知的设置
+                    autoCheckUpdate = bool.Parse(configData.Application.Run.AutoCheckUpdate);
+                    autoGetNotice = bool.Parse(configData.Application.Run.AutoGetNotice);
+                }
+                catch (Exception e)
+                {
+                    // 如果读取或解析配置文件出错，设置默认值并显示错误信息
+                    autoCheckUpdate = true;
+                    autoGetNotice = true;
+                    Console.ForegroundColor = ConsoleColor.Red;
+                    Console.WriteLine($"✕ 读取配置文件时出错:\n{e}\n[!] 请检查配置文件是否正确，您可以先删除配置文件然后运行应用以重新生成默认配置。");
+                    Console.ResetColor();
+                    Environment.ExitCode = 1;
+                    return;
+                }
+            }
+            else
+            {
+                // 如果配置文件不存在，则使用默认设置
+                autoCheckUpdate = true;
+                autoGetNotice = true;
+                Console.ForegroundColor = ConsoleColor.Yellow;
+                Console.WriteLine($"⚠ 没有找到配置文件，将生成默认配置文件！");
+                Console.ResetColor();
+
+                try
+                {
+                    // 创建默认配置对象
+                    var defaultConfig = new ConfigData
+                    {
+                        Information = new Information { Version = "v2.9" },
```

Comment:
```suggestion
                        Information = new Information { Version = "v2.11" },
```

Bump version to v2.11

Reply:
确定？

Bump version to v2.10-1 会不会好点？

Reply:
那先不改